server/chat-plugins/mafia.ts

```diff
@@ -2352,7 +2352,7 @@ export const commands: Chat.ChatCommands = {
 			room = this.requireRoom();
 			const game = this.requireGame(Mafia);
 			if (game.hostid !== user.id && !game.cohostids.includes(user.id)) this.checkCan('mute', null, room);
-			if (game.phase !== 'signups') throw new Chat.ErrorMessage(`Signups are already closed.`);
+			if (game.phase === 'day' || game.phase === 'night') throw new Chat.ErrorMessage(`The game already started.`);
```

Comment:
```suggestion
			if (game.started) throw new Chat.ErrorMessage(`The game already started.`);
```

Started is a simpler way of specifying this.